### PR TITLE
Review changes

### PR DESCRIPTION
**Pull request by Anisha2510**
# Pull request

Implemented review changes suggested by Phil A.

### CODE REVIEW

**Comment by Anisha2510** [comment]
1. Removed the following files
.openapi-generator
.openapi-generator-ignore
mqcloudv1/.openapi-generator-ignore
prepare_project.sh

2. Updated go core version to the latest (5.15.0)
3. README.md:
Modified min supported version of Go to be 1.20 to align with go.mod file.
Fixed the link in the Overview table 

4. .travis.yml:
added 1.21.x to the "go" field
Install version 1.55.2 of golangci-lint instead of 1.51.2 
In the "deploy" stage, use go: 1.20.x 
5. .gitignore:
Changed to Go SDK template repo
In the integration tests, used fmt.Fprintf(GinkgoWriter...) instead of fmt.Println 

Added the .bumpversion.cfg and .releaserc files. 

Recommendations:

README.md:
Changed "MQoC" to "MQ on Cloud"

**Comment by Anisha2510** [comment]
Generated new sdk service with updated api yaml file now default service URL referenced to eu-de specific region.
Also, updated the ITs file generated with new api yaml file.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,6 @@
 
-| Travis |
-| ------ |
-| [![Build Status](https://v3.travis.ibm.com/mqoc/mqcloud-go-sdk.svg?token=xb8AgTFykN966eqXcU5v)](https://v3.travis.ibm.com/mqoc/mqcloud-go-sdk) |
-
-
-# IBM Cloud MQoC Services Go SDK 0.0.1
-Go client library to interact with the various [IBM Cloud MQoC APIs](https://cloud.ibm.com/apidocs).
+# IBM Cloud MQ on Cloud Services Go SDK 0.0.1
```
Comment (shuaibabubakker) [inline README.md:2]: add "Version"
`# IBM Cloud MQ on Cloud Services Go SDK Version 0.0.1 `

File: README.md [hunk 2]
```diff
@@ -1,11 +1,6 @@
...
+Go client library to interact with the various [IBM Cloud MQ on Cloud APIs](https://cloud.ibm.com/apidocs).
```
Comment (shuaibabubakker) [inline README.md:3]: We wanna give our api doc reference which is 

https://cloud.ibm.com/apidocs/mq-on-cloud

but its not available rightnow. Need to check with the team.

File: .golangci.yml [hunk 3]
```diff
@@ -1,15 +1,3 @@
...
-#
-# (C) Copyright IBM Corp. 2020 All Rights Reserved.
-#
-# US Government Users Restricted Rights - Use, duplication or
-# disclosure restricted by GSA ADP Schedule Contract with
-# IBM Corp.
-######################################## {COPYRIGHT-END} ###
-# DO NOT EDIT: Generated by https://github.ibm.com/mqoc/repo-bootstrap
 run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
```
Comment (shuaibabubakker) [inline .golangci.yml:2]: is this file still needed?

i dont find any reference on sdk template repo or sample sdk repos.

worth a check.

File: README.md [hunk 4]
```diff
@@ -23,7 +18,7 @@ Changes might occur which impact applications that use this SDK.
 
 <!-- toc -->
 
-- [IBM Cloud MQoC Services Go SDK 0.0.1](#ibm-cloud-mqoc-services-go-sdk-001)
+- [IBM Cloud MQ on Cloud Services Go SDK 0.0.1](#ibm-cloud-mqoc-services-go-sdk-001)
```
Comment (shuaibabubakker) [inline README.md:21]: this line is not needed

File: README.md [hunk 5]
```diff
@@ -1,11 +1,6 @@
 
-| Travis |
-| ------ |
-| [![Build Status](https://v3.travis.ibm.com/mqoc/mqcloud-go-sdk.svg?token=xb8AgTFykN966eqXcU5v)](https://v3.travis.ibm.com/mqoc/mqcloud-go-sdk) |
-
-
-# IBM Cloud MQoC Services Go SDK 0.0.1
-Go client library to interact with the various [IBM Cloud MQoC APIs](https://cloud.ibm.com/apidocs).
+# IBM Cloud MQ on Cloud Services Go SDK version 0.0.1
```
Comment (shuaibabubakker) [inline README.md:2]: Capital V

**Review by shuaibabubakker — APPROVED** [review]
